### PR TITLE
Adding support for inodes.

### PR DESCRIPTION
**Pull request by vishh**
cc @timstclair

### CODE REVIEW

File: fs/fs.go [hunk 1]
```diff
@@ -395,15 +398,17 @@ func (self *RealFsInfo) GetDirUsage(dir string, timeout time.Duration) (uint64,
 	return usageInKb * 1024, nil
 }
 
-func getVfsStats(path string) (uint64, uint64, uint64, error) {
+func getVfsStats(path string) (uint64, uint64, uint64, uint64, uint64, error) {
```
Comment (timstclair) [inline fs/fs.go]: That's a lot of returns! Either name them, or just return an Fs type

File: fs/fs.go [hunk 2]
```diff
@@ -246,27 +246,30 @@ func (self *RealFsInfo) GetFsInfoForPath(mountSet map[string]struct{}) ([]Fs, er
...
-				total, free, avail uint64
-				err                error
+				err error
+				fs  Fs
 			)
 			switch partition.fsType {
 			case "devicemapper":
-				total, free, avail, err = getDMStats(device, partition.blockSize)
+				fs.Capacity, fs.Free, fs.Available, err = getDMStats(device, partition.blockSize)
+				fs.Type = "devicemapper"
```
Comment (timstclair) [inline fs/fs.go]: Use the constant types here

**Comment by timstclair** [comment]
LGTM, just 2 style nits. Feel free to self merge after fixing.

File: fs/fs.go (hunk 2 above)
Comment (vishh) [inline fs/fs.go]: Fixing

File: fs/fs.go (hunk 1 above)
Comment (vishh) [inline fs/fs.go]: Done

**Comment by vishh** [comment]
Merging. Tests passed after the recent update.

**Pull request merged**